app/src/main/res/values/strings.xml

```diff
@@ -182,9 +182,17 @@
     <string name="playerVideoFormat">Video format for player</string>
     <string name="no_audio">No audio</string>
     <string name="no_video">No video</string>
+    <string name="no_subtitle">No subtitle</string>
     <string name="audio">Audio</string>
     <string name="video">Video</string>
     <string name="downloading">Downloading…</string>
+    <string name="download_paused">Download paused</string>
+    <string name="download_completed">Download completed</string>
```

Comment:
We already have a `downloadsucceeded` string, so the `download_completed` string here shouldn't be needed and instead replaced with the `downloadsucceeded` string resource.